docs/Training-ML-Agents.md

```diff
@@ -151,7 +151,6 @@ environment, you can set the following command line options when invoking
   [here](https://docs.unity3d.com/Manual/CommandLineArguments.html) for more
   details.
 * `--debug`: Specify this option to enable debug-level logging for some parts of the code.
-* `--multi-gpu`: Setting this flag enables the use of multiple GPU's (if available) during training.
```

Comment:
Should this be added to the Migrating.md ?

Reply:
Good call

Reply:
Hopefully it will be re-added before the next release (1.0)

Reply:
Added to changelog. I think the plan is to make the base optimizer(s) do multi-gpu without a separate flag.